.tekton/operator-bundle-pipeline.yaml

```diff
@@ -301,83 +301,107 @@ spec:
       value: "$(params.operator-image-repo):$(tasks.determine-operator-image-tag.results.IMAGE_TAG)"
     taskRef:
       name: wait-for-image
+    # This timeout should be kept in sync with the maximum runtime of the PipelineRun for building the operator image.
+    timeout: 1h10m
```

Comment:
I mentioned this previously (<https://github.com/stackrox/stackrox/pull/13006#discussion_r1801307712>) and I'm still interested in your opinion about this. Should we orient around `spec.timeouts.pipeline` or `spec.timeouts.tasks` for the values here and instructions?

Reply:
My opinion is that basing our timeouts on `spec.timeouts.pipeline` seems reasonable. But since the image pushing would happen as part of the tasks themselves (not as part of the `finally` cleanup), it would actually not matter much(?).